libraries/ag-charts-test/src/canvas/mock-canvas-text.ts

```diff
@@ -10,6 +10,20 @@ export function mockCanvasText() {
     };
 }
 
+const replacements: Partial<Record<string, string>> = {
+    // Math minus
+    '\u2212': '-',
```

Comment:
We didn't technically need it in the end - but if you add a label formatter using `f`, it uses this unicode character for the minus sign (which is correct). I left it in because it might be needed later